cpp/src/io/csv/csv_reader.cu

```diff
@@ -218,8 +228,9 @@ std::string stringType(gdf_dtype dt){
  * 		delim_whitespace	-	use white space as the delimiter - default is false.  This overrides the delimiter argument
  * 		skipinitialspace	-	skip white spaces after the delimiter - default is false
  *
- * 		skiprows			-	number of rows at the start of the files to skip, default is 0
- * 		skipfooter			-	number of rows at the bottom of the file to skip - default is 0
+ *		nrows       	 	-	number of rows of file to read, default is to read all rows
```

Comment:
This whole documentation block needs to be made to be inline with the example here: https://github.com/rapidsai/cudf/blob/24939c7592be3093c5fd213a0d6c91ba6c8c1cb9/cpp/src/examples/documentation_example.cpp